gio/src/subclass/application.rs

```diff
@@ -390,6 +462,54 @@ unsafe extern "C" fn application_handle_local_options<T: ApplicationImpl>(
     imp.handle_local_options(&from_glib_borrow(options)).into()
 }
 
+unsafe extern "C" fn application_dbus_register<T: ApplicationImpl>(
+    ptr: *mut ffi::GApplication,
+    connection: *mut ffi::GDBusConnection,
+    object_path: *const c_char,
+    error: *mut *mut glib::ffi::GError,
+) -> glib::ffi::gboolean {
+    let instance = &*(ptr as *mut T::Instance);
+    let imp = instance.imp();
+
+    match imp.dbus_register(
+        &from_glib_borrow(connection),
+        &glib::GString::from_glib_borrow(object_path),
+    ) {
+        Ok(()) => glib::ffi::GTRUE,
+        Err(e) => {
+            if !error.is_null() {
+                *error = e.into_glib_ptr();
+            }
+            glib::ffi::GFALSE
+        }
+    }
+}
+
+unsafe extern "C" fn application_dbus_unregister<T: ApplicationImpl>(
+    ptr: *mut ffi::GApplication,
+    connection: *mut ffi::GDBusConnection,
+    object_path: *const c_char,
+) {
+    let instance = &*(ptr as *mut T::Instance);
+    let imp = instance.imp();
+    imp.dbus_unregister(
+        &from_glib_borrow(connection),
+        &glib::GString::from_glib_borrow(object_path),
+    );
+}
+
+unsafe extern "C" fn application_name_lost<T: ApplicationImpl>(
+    ptr: *mut ffi::GApplication,
+) -> glib::ffi::gboolean {
+    let instance = &*(ptr as *mut T::Instance);
+    let imp = instance.imp();
+    if imp.name_lost() {
```

Comment:
Use imp.name_lost().into_glib() directly

Reply:
Yeah, right :see_no_evil: I'll also fix the other conditional that broke the build, but probably not before tomorrow evening; it's getting late here :slightly_smiling_face:

Reply:
Fixed.